eval/open_llm_leaderboard/hellaswag_utils.py

```diff
@@ -0,0 +1,24 @@
+import datasets
```

Comment:
This script is also part for the lm_eval repo https://github.com/EleutherAI/lm-evaluation-harness/blob/main/lm_eval/tasks/hellaswag/utils.py
Is this copy actually used and necessary?

Reply:
yes, it does not pick it up from there even though use `include`.